README.md

```diff
@@ -1 +1,123 @@
-# To-Do-List
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [To Do List] <a name="about-project"></a>
+
+**[To Do List]** is project built with webpack, which is a popular open-source JavaScript module bundler that is widely used in modern web development to package and manage assets, such as JavaScript files, CSS styles, images, and more. It helps developers streamline the process of organizing, optimizing, and serving these assets for web applications.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>CSS</li>
+    <li>HTML</li>
+    <li>Javascript</li>
+    <li>Javascript Modules</li>
+    <li>Webpack</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[To do list container]**
+- **[An input field]**
+- **[Dynamically created content]**
+- **[Webpack configuration]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- **[Coming soon]**
+
```

Comment:
- [ ] It would be nice if you remove the Live Demo section since you don't have any working live link 🤔